online-workflow/src/main/java/eu/itesla_project/online/tools/OnlineProcessParameters.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) 2016, All partners of the iTesla project (http://www.itesla-project.eu/consortium)
+ * Copyright (c) 2016, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package eu.itesla_project.online.tools;
+
+import org.joda.time.DateTime;
+
+/**
+*
+* @author Giovanni Ferrari <giovanni.ferrari@techrain.it>
+*/
+public class OnlineProcessParameters{
+
+	private String caseType;
+	private Integer states;
+	private String name;
+	private String owner;
+	private DateTime creationDate;
+	private DateTime date;
+	private String basecasesInterval;
```

Comment:
Verify if these parameters could become final
Verify if these parameters could really be null